data/json/monsters/nether.json

```diff
@@ -565,7 +565,7 @@
     "armor_bash": 6,
     "armor_cut": 12,
     "armor_bullet": 10,
-    "harvest": "demihuman",
+    "harvest": "demihuman_snail",
     "special_attacks": [ [ "ACID", 15 ] ],
     "anger_triggers": [ "PLAYER_WEAK", "FRIEND_DIED" ],
     "flags": [ "SMELLS", "HEARS", "POISON", "ACIDPROOF", "ACID_BLOOD" ]
```

Comment:
I just realized - apparently the eyestalks don't have functional eyes?

Reply:
In game or irl?

Reply:
IRL, they do. This critter has neither vision ranges specified nor SEES as a flag?

Reply:
That probably needs a fix.